src/components/Registration/RegAddEditOrgPerson.vue

```diff
@@ -334,8 +359,8 @@ export default class RegAddEditOrgPerson extends Mixins(AddEditOrgPersonMixin) {
 .lookup-toggle {
   font-size: $px-14;
   text-decoration: underline;
-  pointer-events: none;
-  opacity: 0.4;
+  // pointer-events: none;
+  // opacity: 0.4;
```

Comment:
Normally we should delete obsolete code but in this case the UI/UX team might want me to disable these elements again. (It's happened before.)